ccd-definition/CaseEvent.json

```diff
@@ -447,13 +447,27 @@
     "Name": "Add statement of service (c9)",
     "Description": "Add statement of service",
     "DisplayOrder": 11,
-    "PreConditionState(s)": "Submitted",
-    "PostConditionState": "Submitted",
+    "PreConditionState(s)": "Gatekeeping",
+    "PostConditionState": "Gatekeeping",
```

Comment:
Hmm a bit surprised why this change is here. Why do we need to change the state of add statement of service to gatekeeping?

Reply:
We had a chat with Gail and she decided that it should be moved and was only in submitted before because the gatekeeping state didn't exist. We can double check with Gail before actually doing the PR.